numpy/linalg/lapack_litemodule.c

```diff
@@ -17,6 +17,8 @@ typedef struct { float r, i; } f2c_complex;
 typedef struct { double r, i; } f2c_doublecomplex;
 /* typedef long int (*L_fp)(); */
 
+int FNAME(MAIN_)() { return 0; }
```

Comment:
I think you need a `void` inside the parenthesis to silence the compiler warnings and make Travis happy.